### ISSUE TITLE
wrong dmx out, like it's confmusing/mixing the steps

### ISSUE THREAD

**Issue by SamiNeverland** [mtongnz/espDMX#9]
Greetings everyone, 
thank you all for your contributions to DMX.

I 'm playing with the espDMX example and i have some issues running the code on wemos D1 mini v2 and i have implemented the MAX485 topology on dmxA(Tx0), a status led and i connected a 7 channel RGB led par.

I modified the steps for an R-G-B-black:

1-Master Dimmer
2-Red
3-Green
4-Blue
5-strobo
6-macro1
7-macro2

`// 4 steps with 7 channels in each
byte dmxChase[][7] = { { 255, 0, 0, 0, 0, 0, 0},
                        { 255, 255, 0, 0, 0, 0, 0},
                        { 255, 0, 255, 0, 0, 0, 0},
                        { 255, 0, 0, 255, 0, 0, 0}
 
};
void loop() {
  for (int i = 0; i < 5; i++) {
    
    // Output channels 1 - 10 on dmxA
    dmxA.setChans(dmxChase[i], 7, 1);

    // 1 second between each step
    delay(1000);
  }
}
`
what i get as a result to the led par is:
at the very first loop is almost the sequence as i wrote BUT
after that, every time the status led is blinking, it's like it runs 5 steps.

Any ideas?
Thanks in advance

**Comment by Kokichiro** [comment]
Maybe

**Comment by mtongnz** [comment]
Agreed. Some fixtures don't follow dmx spec and this can cause issues
